src/recipe/Baked.Recipe.Service.Application/Database/PostgreSql/PostgreSqlDatabaseFeature.cs

```diff
@@ -15,13 +15,17 @@ public void Configure(LayerConfigurator configurator)
             services.AddSingleton<ITransaction, FlatTransaction>();
         });
 
+        configurator.ConfigureFluentConfiguration(fluent =>
+        {
+            if (_autoUpdateSchema)
+            {
+                fluent.UpdateSchema();
```

Comment:
remove optional parameters and default values, and give values explicitly from here. this will make it more readable.